modules/appeals_api/app/serializers/appeals_api/evidence_submission_serializer.rb

```diff
@@ -1,13 +1,30 @@
 # frozen_string_literal: true
 
+require_dependency 'common/exceptions'
 require 'fast_jsonapi'
 
 module AppealsApi
-  class EvidenceSubmissionSerializer
-    include FastJsonapi::ObjectSerializer
+  class EvidenceSubmissionSerializer < ActiveModel::Serializer
```

Comment:
Can you help me understand the goal behind this change?

I worry about moving away from a non-standard implementation, considering the cost of changing in the future.

Reply:
How is this a non-standard implementation?

Reply:
The rest of vets-api uses FastjsonApi. If we diverge from that, we're supporting a unique implementation of serialization. When changes to serialization happen, we'll need to update our serialization individually

Reply:
![image](https://user-images.githubusercontent.com/30485143/114043281-262b9f80-9854-11eb-8460-d9c37f718d6e.png)
this is just a sample of the serializers in vets-api using AM Serializer as opposed to Fastjson